### PR TITLE
Transfer repo to dwyl simplify readme intro and focus on WIIFM

### PR DESCRIPTION
**Pull request by nelsonic**
This is _only_ **the _beginning_**. 
We _know_ ***exactly*** what we are doing in 2017!

@iteles please review/correct and when happy, merge. thanks! ❤️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,248 +1,416 @@
-# Practical Full-Stack JavaScript Web Application Test Driven Development
+# The _Practical_ "Book"
+## on _Crafting_ Full-Stack Web Applications
 
-A ***practical guide*** to ***Full Stack*** ***T***est ***D***riven ***D***evelopment (TDD) for people building ***robust*** **node.js** **web / mobile** applications in teams!
+A ***step-by-step guide*** to "***Full Stack***" Web Application Development
+for people building ***beautiful***, ***robust*** & ***maintainable***
+**web / mobile** app individually or in teams!
```
Comment (iteles) [inline README.md]: app*s*

File: README.md [hunk 2]
```diff
@@ -1,248 +1,417 @@
...
+### "*Front End*"?
+
+There is ***one (programming) language*** that *all* web
+browsers "understand": ***JavaScript***.  
+As a result it is the _most popular programming language_
+(_see StackOverflow survey results above_!)
+
+Additionally you will need to know two other "languages": HTML and CSS ... but:
+
+![stay calm](https://cloud.githubusercontent.com/assets/194400/21954158/cea1b17a-da41-11e6-9f0e-a73627125934.png)
```
Comment (iteles) [inline README.md]: Feels like it's missing something - "I wasn't thinking of panicking but now you're telling me not to... why shouldn't I?"

I was expecting to know why I shouldn't be worried in the sentence after the picture.

File: README.md [hunk 3]
```diff
@@ -1,248 +1,417 @@
...
+If you are _still_ not writing tests _**why not**_?
+We _urge_ you to _read_ the *first* chapter ("_A Brief Introduction to TDD_") and decide for yourself if you _still_ think its "_too difficult_" or "_takes longer_".
+-->
 
+### "_Free_" and Open Source...?
 
-We have better phones; shouldn't we also have better software development methods?
+Yes, the book is **100% _Free_ and _Open Source_**  on GitHub for anyone to
+read and [contribute](#contribute) to!    
+We (*strongly*) believe that ***all learning materials*** should be
```
Comment (iteles) [inline README.md]: I know what you mean by this, but we don't want to give the impression (given the massive library we have) that we expect people to pour months of their life into creating quality content for *free*.
We need to convey that we know and acknowledge the value of people's time (which is why we pay github even though all our repos are open source for example) whilst still detailing our reasons for open sourcing ours.
I know there's a little on this further down, but this is the kind of sentence that causes a reaction big enough that people will stop reading and just *whine.*

File: README.md [hunk 4]
```diff
@@ -1,248 +1,417 @@
...
+## Star the Project :star:
 
-## Spreading the Word >> *Please Re-Tweet*!
+The _easiest_ way you can _contribute_ is
+by "_starring_" this project on GitHub!
+This will help you to "_bookmark_" the content so you can return to it.
+But it will also help the people who "_follow_" you on GitHub to _discover_
+that you find it _interesting_ or _useful_.
+
+The _more_ people star and _share_ the project, the more possible contributors are able to understand the value of contributing and *open sourcing their knowledge*!
```
Comment (iteles) [inline README.md]: I finished this sentence (by adding what is after the comma), no idea if this was your original intention for the sentence.

**Comment by iteles** [comment]
🎉 🙌 Just finished reading, excited for the fresh start!

I pushed a commit:
+ Fixing typos
+ Changing two tiny things in the flow
+ Removing the small sprinkling of `<br/>` elements in favour of plain `markdown` (i.e. Adding 2 spaces at the end of the line and hitting enter to create the line break)

I haven't touched the content itself.

Other than the comments I have added on specific lines, my only suggestion would be that we can evolve it to be even more concise and to-the-point. If you can give me a bullet point list of what you would like to get across, I can review this with an 'editor's eye' 👀 and make some effective cuts to the wording whilst making the key points stand out for people skim-reading. 👍  (in a separate PR)

File: README.md (hunk 3 above)
Comment (nelsonic) [inline README.md]: Agreed. Thanks.

File: README.md (hunk 2 above)
Comment (nelsonic) [inline README.md]: Please creat an issue to expand/clarify that section thanks.

**Comment by nelsonic** [comment]
The _reason_ for using the "<br />" ***instead*** of the double space markdown is the sheer number of PRs we've had in the past two years in which people "remove trailing white space"... it needs to be ***explicit*** else people will remove the syntactic content breaks.

**Comment by iteles** [comment]
@nelsonic Agreed, I get that, I'm currently opening the issues from this PR and adding `<br/>`s consistently is one of them.

**Comment by nelsonic** [comment]
@iteles _sweet_! 🍭  😍

**Pull request merged**